shell/platform/tizen/flutter_project_bundle.h

```diff
@@ -70,6 +75,9 @@ class FlutterProjectBundle {
   // Path to the AOT library file, if any.
   std::filesystem::path aot_library_path_;
 
+  // Custom entrypoint in the Dart project
+  std::string custom_dart_entrypoint_;
```

Comment:
What is this for?

Reply:
A most typical scenario is a multi-packaging app. 
this is not a new feature, it has just been refactored. please see [this](https://github.com/flutter-tizen/engine/blob/fc30c34c4d8542d62876080dc4c9a3b2e01f8527/shell/platform/tizen/flutter_tizen.cc#L62)